src/main/java/net/fabricmc/loader/impl/transformer/FabricTransformer.java

```diff
@@ -30,7 +30,8 @@ public static byte[] transform(boolean isDevelopment, EnvType envType, String na
 		boolean isMinecraftClass = name.startsWith("net.minecraft.") || name.startsWith("com.mojang.blaze3d.") || name.indexOf('.') < 0;
 		boolean transformAccess = isMinecraftClass && FabricLauncherBase.getLauncher().getMappingConfiguration().requiresPackageAccessHack();
 		boolean environmentStrip = !isMinecraftClass || isDevelopment;
-		boolean applyAccessWidener = isMinecraftClass && FabricLoaderImpl.INSTANCE.getAccessWidener().getTargets().contains(name);
+		boolean applyAccessWidener = FabricLoaderImpl.INSTANCE.getGameProvider().isAccessWidenable(name)
```

Comment:
This will not compile because `isAccessWidenable()` is not a method in GameProvider.